content/docs/pipelines/sdk/enviroment_variables.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Using enviroment varibales in piplines"
+description = "How to set and use enviroment variables in kubeflow piplines"
++++
+
+This page describes how to create a enviroment varaibale for Kubeflow Pipelines 
+and how they work in a pipline. 
+
+## Before you start
+
+### Initial setup
+Before starting with this tutorial the following things have to be in place: 
```

Comment:
Suggested revision:

Set up your environment(Shorter wording, use general KF installation page):

* [Install Kubeflow](/docs/started/getting-started/)
* [Install the Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/)